CONTRIBUTING.md

```diff
@@ -32,23 +39,23 @@ Our base of operations for group discussions on this project: [Jupiter Web Site
 
 * 🚀 The current focus and priority is on [Milestone 1.0](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/milestone/1)
 * 📝 You can submit an idea or a feature by opening an [issue](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues).
-  * _Please do a quick search prior - maybe a relevant issue already exists_ 😉
-* 💡 When you feel inspired about a particular issue or feature, please propose your implementation idea/strategy for approval via  comments, which indicates you are dedicated to solving this issue, thus avoiding duplication of effort.
+* _Please do a quick search prior top opening an issue - maybe a relevant issue already exists_ 😉
```

Comment:
I think this should be to rather than top.

Reply:
Ok done, thanks!